src/routes/api/installation/api-installation-syncstate-get.ts

```diff
@@ -4,9 +4,32 @@ import { Subscription } from "models/subscription";
 import { pick } from "lodash";
 
 export const ApiInstallationSyncstateGet = async (req: Request, res: Response): Promise<void> => {
+	if (req.query.limit === undefined) {
+		const msg = "Missing limit query param";
+		req.log.warn({ req, res }, msg);
+		res.status(400).send(msg);
+		return;
+	}
+
+	if (req.query.offset === undefined) {
+		const msg = "Missing offset query param";
+		req.log.warn({ req, res }, msg);
+		res.status(400).send(msg);
+		return;
+	}
+
 	const githubInstallationId = Number(req.params.installationId);
 	const gitHubAppId = Number(req.params.gitHubAppId) || undefined;
 	const jiraHost = req.params.jiraHost;
+	const limit = Number(req.query.limit);
+	const offset = Number(req.query.offset);
+
+	if (limit > 10000) {
```

Comment:
nit 10000 coming from config better.  At least make it a constant value

Reply:
just an admin endpoint we probably never used, I woudn't bother much

Reply:
turned into a constant though :) thanks!